packages/features/ee/organizations/components/CreateANewOrganizationForm.tsx

```diff
@@ -119,10 +138,15 @@ export const VerifyCodeDialog = ({
                     setError("The code is a required field");
                   } else {
                     setIsLoading(true);
-                    verifyCodeMutation.mutate({
-                      code: value,
-                      email,
-                    });
+                    isUserSessionRequiredToVerify
```

Comment:
we require only logged in users to create organisation so i'll undo the changes in this file.